src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -94,6 +99,7 @@ object EasyStageDataset {
 
     def createFileSdo(file: File, parentSDO: String): Try[Unit] = {
       log.debug(s"Creating file SDO for $file")
+      val r = getUserId
```

Comment:
Yes.. you're right... we refer to different lines... I though about the "createFileSdo" method, after I wrote the comment, I realised that.
I've changed it. Thanks.